### DISCUSSION TITLE
[help] harvester

### DISCUSSION THREAD

**Discussion by Alissonverd** [Chia-Network/chia-blockchain#4617] [Support]
It's possible to run more than one harvester per computer using Linux?

**Comment by sargonas** [comment]
Not to my knowledge, nor would you want to as far as I can see. The harvester instance is going to bind to a set number of ports and to a single Daemon. once that happens other harvester instances are going to conflict because the ports are taken. You could, in theory, run a new instance configured for other ports out of a separate install folder, but they would fight over the same daemon.

Is there a particular problem you are trying to solve for with running two harvesters on one system? If so maybe I can help you find a more appropriate solution.

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
